src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,27 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
         List<Product> products = getAllProductsFromShoppingCart(userId);
         Order order = new Order(products, userId);
-        // NOTE: In production ready code this order identifier should be generated by DB
-        // For test purpose we simplify this and return dummy data
+        logger.debug("Successfully ordered products by user: userId{};", userId);
         order.setOrderId(1L);
         return order;
     }
 
     private List<Product> getAllProductsFromShoppingCart(Long userId) {
-        // NOTE: In production ready code this method should fetch data from DB
-        // For test purpose we simplify this method and return dummy data
         Product iphone = new Product("iPhone X", BigDecimal.valueOf(1199));
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        logger.debug("Successfully fetched data from DB by user : usedId{};", userId);
```

Comment:
```suggestion
        logger.info("Successfully fetched data from DB by user : usedId{};", userId);
```
debug level is used when indicating that some method was called or in other debugging issues
info level is used to track business logic. here it is important for business logic that products were fetched from DB